1.30.0: compiltion error on Debian

**1. What were you trying to do?**

Build a Debian package for vg v1.30.0

**2. What did you want to happen?**


**3. What actually happened?**
```
. ./source_me.sh && g++ -I/usr/include/fastahack -I/usr/include/smithwaterman -I/usr/include/vcflib -I/usr/include/smithwaterman -I/usr/include/fastahack -I/build/vg-1.30.0+ds/include -isystem /build/vg-1.30.0+ds/include -I. -I/build/vg-1.30.0+ds/src -I/build/vg-1.30.0+ds/src/unittest -I/build/vg-1.30.0+ds/src/subcommand -I/build/vg-1.30.0+ds/include/dynamic -pthread -isystem /usr/include/cairo -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/pixman-1 -isystem /usr/include/uuid -isystem /usr/include/freetype2 -isystem /usr/include/libpng16 -Wdate-time -D_FORTIFY_SOURCE=2 -O3 -Werror=return-type -std=c++14 -ggdb -g -g -O2 -ffile-prefix-map=/build/vg-1.30.0+ds=. -fstack-protector-strong -Wformat -Werror=format-security -DSIMDE_ENABLE_OPENMP -fopenmp-simd -O3 -mavx2 -fopenmp -MMD -MP -c -o obj/index_registry.o src/index_registry.cpp 
In file included from src/index_registry.cpp:3:
src/index_registry.hpp:207:57: error: expected class-name before ‘{’ token
  207 | class InsufficientInputException : public runtime_error {
      |                                                         ^
src/index_registry.cpp: In constructor ‘vg::InsufficientInputException::InsufficientInputException(const string&, const vg::IndexRegistry&)’:
src/index_registry.cpp:1235:5: error: type ‘std::runtime_error’ is not a direct base of ‘vg::InsufficientInputException’
 1235 |     runtime_error("Insufficient input to create " + target), target(target), inputs(registry.completed_indexes())
      |     ^~~~~~~~~~~~~

```

**4. If you got a line like `Stack trace path: /somewhere/on/your/computer/stacktrace.txt`, please copy-paste the contents of that file here:**

```
Place stacktrace here.
```

**5. What data and command can the vg dev team use to make the problem happen?**

I can make a Dockerfile that reproduces this (the Debian build process), if desired.

**6. What does running `vg version` say?**

```
Place vg version output here
```

--- a/src/index_registry.hpp
+++ b/src/index_registry.hpp
@@ -7,6 +7,7 @@
 #include <functional>
 #include <string>
 #include <memory>
+#include <stdexcept>
 
 namespace vg {
 

